### ISSUE TITLE
Better alt text for Knight Foundation Logo

### ISSUE THREAD

**Issue by seanredmond** [nypl-spacetime/nypl-spacetime.github.io#11]
Alt text for the logo should be changed to "Knight Foundation." As it is now, a screen reader this as "With support by Knight Foundation logo."
